lib/manager/index.ts

```diff
@@ -44,6 +45,14 @@ function validateManager(manager): boolean {
 
 const managers = loadModules<ManagerApi>(__dirname, validateManager);
 
+managers.custom = {
+  defaultConfig: {
+    autoReplace: true,
+    versioning: 'semver',
+  },
+  extractPackageFile: customManager.extractPackageFile,
+};
+
```

Comment:
Why this? 🤔

Reply:
I thought about adding code to `lib/manager/custom/*` but it would then require some changes to the loading of modules, and I decided this is cleaner because it's very much a "one off" type of manager.

Reply:
I could try putting this code into `lib/manager/custom/*` and seeing how any `if (manager !== 'custom')` workarounds are needed

Reply:
ok, I thought it could beloaded as normal manager.

Reply:
It maybe can, but it needs a custom for-loop once it comes to fileMatching and so on. Essentially if you have X customerManagers definitions then it's like having X additional managers